_posts/2020-02-10-week-2.md

```diff
@@ -30,6 +30,14 @@ View our detailed lesson plan [here](https://drive.google.com/open?id=13VB839Yxd
 - **Worksheet** (25 min): Log into Scratch accounts and work on the worksheet
 - **Wrap up** (5 min): See how conditionals fit with the maze activity
 
+### Video Materials
```

Comment:
change intro: We've produced exciting videos that go over the topics outlined in our lesson plan. These can serve as inspiration for how you could conduct your lessons. If you or the students don’t have time to watch the videos...check out our quick cheat sheet: https://drive.google.com/file/d/1ndrnZxA1OVSXp6b4z0ZJV3ePsPDaQmXC/view?usp=sharing